.gitignore

```diff
@@ -1,4 +0,0 @@
-# build directories
-build/
```

Comment:
It was for me useful. Why do you want to remove this ?

Reply:
https://cmake.org/cmake/help/latest/manual/cmake.1.html discourages these kinds of in source builds. If you are using an IDE that does an in-source build anyway, the recommended way would be to use a global git ignore (set in ~/.gitconfig > core.excludesFile) to filter out all IDE specific files across all git projects on the computer once and for all

Reply:
ok thx